src/Cell.cpp

```diff
@@ -119,16 +244,23 @@ std::map<int, surface_halfspace> Cell::getSurfaces() const {
 
 
 /**
- * @brief Set the ID for the Universe within which this Cell resides.
- * @param universe the Universe's user-specified ID
+ * @brief Sets the name of the Cell
+ * @param name the Cell name string
  */
-void Cell::setUniverse(int universe) {
-  _universe = universe;
+void Cell::setName(const char* name) {
+  int length = strlen(name);
+
+  /* Initialize a character array for the Cell's name */
+  _name = new char[length+1];
```

Comment:
Good catch! I just committed a fix for the ``Cell``, ``Surface``, ``Universe``, ``Lattice``, and ``Material`` classes.